flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/plan/abilities/source/WatermarkPushDownSpec.java

```diff
@@ -104,11 +121,25 @@ public String getDigests(SourceAbilityContext context) {
                         watermarkExpr,
                         JavaScalaConversionUtil.toScala(
                                 context.getSourceRowType().getFieldNames()));
-        if (idleTimeoutMillis > 0) {
-            return String.format(
-                    "watermark=[%s], idletimeout=[%d]", expressionStr, idleTimeoutMillis);
+        String digest = String.format("watermark=[%s]", expressionStr);
+        long actualIdleTimeoutMillis = calculateIdleTimeoutMillis();
+        if (actualIdleTimeoutMillis > 0) {
+            digest = String.format("%s, idletimeout=[%d]", digest, actualIdleTimeoutMillis);
```

Comment:
```suggestion
            digest = String.format("%s, idleTimeout=[%d]", digest, actualIdleTimeoutMillis);
```

Reply:
The original name is `idletimeout`, it needs to be consistent with the original name